3002.09 <c-41#13> Created logger
3002.1 <c-41#13:21> Logger was updated
3002.1 <c-41#13:21> Started intializing cube lib
3002.1 <c-41#13:21> Finished intializing cube lib with worker
3002.21 <c-41#13:21> shareCubes is called
3002.21 <c-41#13:21> Added 0 failed local cubes
3002.21 <c-41#13:21> Pruned 0 local cubes
3002.21 <c-41#13:21> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
3002.21 <c-41#13:21> returnFailedAndRequestCubes => [2] {}
3002.21 <c-41#13:21> Sent 0 failed cubes to root and requested new cubes
3002.21 <c-41#13:21> Received 3 cubes from root
3002.21 <c-41#13:21> CubeSolverThread entered cube retrieval loop, currently there are 3 cubes
3002.21 <c-41#13:21> CubeSolverThread retrieved 3 cubes
3002.21 <c-41#13:21> Started solving a cube
3002.22 <c-41#13:21> returnFailedAndRequestCubes => [2] {}
3002.22 <c-41#13:21> Sent 0 failed cubes to root and requested new cubes
3002.22 <c-41#13:21> Received 3 cubes from root
3002.43 <c-41#13:21> Cube failed
3002.43 <c-41#13:21> Added failed cube
3002.43 <c-41#13:21> Started solving a cube
3002.43 <c-41#13:21> Cube failed
3002.43 <c-41#13:21> Added failed cube
3002.43 <c-41#13:21> Started solving a cube
3173.96 <c-41#13:21> appl_interrupt was called
3173.96 <c-41#13:21> appl_interrupt was called
3173.96 <c-41#13:21> appl_withdraw was called
3173.96 <c-41#13:21> Started withdraw thread
3173.96 <c-41#13:21> STOP (0.00s since last cb)
3173.96 <c-41#13:21> Solving interrupted
3173.96 <c-41#13:21> Leaving the main loop
3173.98 <c-41#13:21> Enter destructor of CubeWorkerInterface
3173.98 <c-41#13:21> Enter destructor of CubeRoot
3173.98 <c-41#13:21> Finished withdraw thread
3235.22 <c-41#13:21> Enter destructor
3235.22 <c-41#13:21> Time working: 171.862
3235.22 <c-41#13:21> Time suspended: 0.000
3235.22 <c-41#13:21> Joined cleanup thread
3235.22 <c-41#13:21> Exit destructor
3235.22 <c-41#13:21> Destructing logger
